apple-pie/styles/style.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+
+/* main{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+} */
+body {
+    font-family: Poppins;
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    width: 100%;
+}
+body > section { 
+    margin: 50px 100px 50px 100px;
+}
+
+p {
+    font-size: 2em;
+    text-justify: auto;
+}
+
+
+.banner-image {
+    width: 100%;
+    height: 700px;
+    object-fit: cover;
+}
+
+h1 {
+    font-size: 5em;
+}
+
+h2 {
+    font-size: 2em;
+    font-weight: 900;
+}
+/* .centered {
+        position: absolute;
+        top: 40%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+} */
+
+.container {
+    position: relative;
+    text-align: center;
+    color: white;
+    background-image: url('../images/apple-pie.jpg');
+    background-size: cover;
+    height: 700px;
+    width: 100vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+   }
+
+.aligned-elements {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: space-around;
+	align-items: center;
+	align-content: space-between;
+}
+
```

Comment:
you can add it to limit the size of the images inside the .aligned-elements div
`.aligned-elements img {
  max-width: 100vw;
}`